### PR TITLE
[Feature] Update document & keep connectors

### PR DESCRIPTION
**Pull request by Dvergar**
Can be tested against https://studio.chiligrafx-dev.com/?workspace=latest&engine=356

We can now keep connector states betwen document loads.

## Related tickets

https://chilipublishintranet.atlassian.net/browse/EDT-933

### CODE REVIEW

**Review by pkgacek — APPROVED** [review]

**Review by Matthiee — APPROVED** [review]

**Review by brapoprod — APPROVED** [review]

**Review by Matthiee — APPROVED** [review]

File: src/types/DocumentTypes.ts [hunk 1]
```diff
@@ -198,3 +198,7 @@ export type TemplateDocumentProperties = DocumentProperties;
 export interface ProjectDocumentProperties extends DocumentProperties {
     templateId: string;
 }
+
+export type DocumentLoadOptions = {
+    keepConnectors: boolean;
```
Comment (NicolaVerbeeck) [inline src/types/DocumentTypes.ts:203]: Should we add the docs describing each option in here too?

**Review by NicolaVerbeeck — DISMISSED** [review]

File: src/types/DocumentTypes.ts (hunk 1 above)
Comment (brapoprod) [inline src/types/DocumentTypes.ts:203]: should be optional

Comment (Dvergar) [reply src/types/DocumentTypes.ts:203]: @brapoprod WDYT Brian?

It's not done for the other ones and it's still available in `load` doc atm.

**Review by brapoprod — APPROVED** [review]

File: src/types/DocumentTypes.ts (hunk 1 above)
Comment (brapoprod) [reply src/types/DocumentTypes.ts]: I indeed think it makes most sense on top of the load function

**Review by srares76 — APPROVED** [review]

**Review by NicolaVerbeeck — APPROVED** [review]

**Pull request merged**